.github/workflows/actions.yaml

```diff
@@ -0,0 +1,45 @@
+name: Code Quality Checks
```

Comment:
good job with additional task: pull_request + push, eslint + prettier
hope eslint and prettier will be never conflicting though)